### PR TITLE
Hide eater section on eaters contact page

### PR DESCRIPTION
**Pull request by robinkeunen**
- Tache [s0058 - Imprimer cartes des mangeurs](https://odoo.bees-coop.be/web#id=288&view_type=form&model=project.task&action=351&active_id=2)
- [specifications](https://docs.google.com/document/d/1TEtB4A40kfetGswxJ2B9erf1mGxu1ZgUklfIOyxCjZU/edit?usp=sharing)

### CODE REVIEW

**Review by remytms — COMMENTED** [review]
There is a typo error in your commit message for a6d1817 . :)
hive -> hide

File: beesdoo_base/report/beescard.xml [hunk 1]
```diff
@@ -58,27 +58,27 @@
...
-                                    style="font-size: 0.8em;font-family:Roboto-Regular">
-                                    <t t-if="partner.parent_eater_id and partner.parent_eater_id.cooperator_type == 'share_a'">
-                                        <span
-                                            t-raw="'[' + (partner.parent_eater_id.first_name and partner.parent_eater_id.first_name[:30] or '') + ' ' + (partner.parent_eater_id.last_name and partner.parent_eater_id.last_name[:30] or '') + ']'"
-                                            style="font-family:Roboto-Regular" />
+                                <p style="text-align: left;padding-top: 0.2cm;font-size: 1em;">
+
+                                    <t t-if="partner.parent_eater_id and partner.eater == 'eater'">
+                                        <!--fix here -->
+                                        <span t-raw="partner.parent_eater_id.first_name and partner.parent_eater_id.first_name[:30] or ''" style="font-family:Roboto-Regular" />
```
Comment (houssine78) [inline beesdoo_base/report/beescard.xml]: missing semi-colon to the end of style parameter

**Comment by houssine78** [comment]
except one remark 👍

**Comment by robinkeunen** [comment]
- reworded commit e43f664
- added style line.

If ok with @houssine78 and @remytms, I'll rebase --autosquash and merge.

**Comment by remytms** [comment]
Runbot have to be green before merging: http://runbot.bees-coop.be/runbot/ :)

**Pull request merged**